app/assets/stylesheets/homepage.css.scss

```diff
@@ -38,8 +38,13 @@
     padding: 9px 12px 0 12px;
 }
 
+.nav-bar-link, .nav-bar-link:visited, .nav-bar-link:link, #logo, #logo:visited, #logo:link {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Each selector in a comma sequence should be on its own line